releases/v20.2.0-beta.2.md

```diff
@@ -0,0 +1,103 @@
+---
+title: What&#39;s New in v20.2.0-beta.2
+toc: true
+summary: Additions and changes in CockroachDB version v20.2.0-beta.2 since version v20.2.0-beta.1
+---
+
+## September 24, 2020
+
+Get future release notes emailed to you:
+
+<div class="hubspot-install-form install-form-1 clearfix">
+    <script>
+        hbspt.forms.create({
+            css: '',
+            cssClass: 'install-form',
+            portalId: '1753393',
+            formId: '39686297-81d2-45e7-a73f-55a596a8d5ff',
+            formInstanceId: 1,
+            target: '.install-form-1'
+        });
+    </script>
+</div>
+
+### Downloads
+
+<div id="os-tabs" class="clearfix">
+    <a href="https://binaries.cockroachdb.com/cockroach-v20.2.0-beta.2.darwin-10.9-amd64.tgz"><button id="mac" data-eventcategory="mac-binary-release-notes">Mac</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v20.2.0-beta.2.linux-amd64.tgz"><button id="linux" data-eventcategory="linux-binary-release-notes">Linux</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v20.2.0-beta.2.windows-6.2-amd64.zip"><button id="windows" data-eventcategory="windows-binary-release-notes">Windows</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v20.2.0-beta.2.src.tgz"><button id="source" data-eventcategory="source-release-notes">Source</button></a>
+</div>
+
+### Docker image
+
+{% include copy-clipboard.html %}
+~~~shell
+$ docker pull cockroachdb/cockroach-unstable:v20.2.0-beta.2
+~~~
+
+
+### Security updates
+
+- A new experimental flag `--accept-sql-without-tls` has been introduced for [`cockroach start`](cockroach-start.html) and [`cockroach-start-single-node`](cockroach-start-single-node.html). When specified, a secure node will also accept secure SQL connections without TLS. When this flag is enabled:
+
+  - Node-to-node connections still use TLS: the server must still be started with `--certs-dir` and valid [TLS cert configuration](authentication.html) for nodes. 
+  - Client [authentication](authentication.html) (spoof protection) and [authorization](authorization.html) (access control and privilege escalation prevention) are performed by CockroachDB as usual, subject to the HBA configuration (for AuthN) and SQL privileges (for AuthZ). 
```

Comment:
AuthN and AuthZ > authentication and authorization (fun fact: AuthN and AuthZ are Raphael terms, not technical terms 😄 )